frontend/src/App.tsx

```diff
@@ -1,13 +1,19 @@
 import Tasks from "../components/Task.js";
+import Cookie from "../components/Cookie.js";
+import Header from "../components/Header.js";
+import { useState } from "react";
 import "./App.css";
 
 function App(): JSX.Element {
+  const [List, setList] = useState<string[]>([]);
+
   return (
     <>
+      <Cookie />
       <h1>Hello world</h1>
-
       <div className="task-box">
-        <Tasks />
+        <Header list={List} setter={setList} />
```

Comment:
Avoid generic names like `list` if it's always going to be a list of the same stuff, in this case of tasks. Don't be afraid of doing `  const [Tasks, setTasks] = useState<string[]>([]);`, `<Header tasks={Tasks} setTasks={setTasks} />` ... - repeating the same name is not wrong if you give it to the same thing

Reply:
Noted, and changed.